posthog/api/organization.py

```diff
@@ -132,6 +131,14 @@ def get_setup(self, instance: Organization) -> Dict[str, Union[bool, int, str, N
             "has_invited_team_members": instance.invites.exists() or instance.members.count() > 1,
         }
 
+    def get_teams(self, instance: Organization) -> List[Dict[str, Any]]:
+        visible_teams = (
```

Comment:
This seems inefficient and it's an endpoint we call on every app load, I think we could accomplish this with a single query. We could have a similar method to `get_effective_membership_level` that already appends the effective level to each team in the list.

Reply:
It's still inefficient but now a bit less so indeed, by reusing the fact that `effective_membership_level` has to be serialized anyway.